src/app/schools/schoolWithRelation.ts

```diff
@@ -0,0 +1,32 @@
+import {ChildSchoolRelation} from '../children/childSchoolRelation';
+import {School} from './school';
+
+export class SchoolWithRelation {
```

Comment:
Same discussion point as for `ChildWithRelation`: It would be easier to understand if there is just a `School` class that integrates this directly in my opinion.